config/manager/nephe-controller.yaml

```diff
@@ -17,6 +17,15 @@ spec:
   selector:
     control-plane: nephe-controller
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nephe-config
+  namespace: system
+data:
+  nephe-controller.conf: |
+    ResourcePrefix: nephe-
```

Comment:
This should be in under '#' and add a comment description indicating what this field does

Reply:
Solved